src/Microsoft.AspNet.Authentication/DataHandler/Serializer/TicketSerializer.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNet.Authentication.DataHandler.Serializer
 {
     public class TicketSerializer : IDataSerializer<AuthenticationTicket>
     {
-        private const int FormatVersion = 2;
+        private const int FormatVersion = 3;
```

Comment:
Are you relying on this to determine the correct deserialization routine to use?
If so, this will not work in a heterogeneous where some servers are upgraded and some are not.